bigchaindb/commands/bigchain.py

```diff
@@ -9,12 +9,20 @@
 import argparse
 import copy
 import json
+import builtins
+
+import logstats
+
 
 import bigchaindb
 import bigchaindb.config_utils
+from bigchaindb.util import ProcessGroup
+from bigchaindb.client import temp_client
 from bigchaindb import db
-from bigchaindb.exceptions import DatabaseAlreadyExists, KeypairNotFoundException
-from bigchaindb.commands.utils import base_parser, start
+from bigchaindb.exceptions import (StartupError,
```

Comment:
This is something that I saw in another pull request - how to do multi line imports.
Now its a good time to agree on a best practice and add it to the python style guide.

Another solution is:
```python
from bigchaindb.exceptions import StartupError, DatabaseAlreadyExists
from bigchaindb.exceptions import KeyPairNotFoundException
```
This approach seems simpler to read but if there is code that is executed at a module level, it will be executed multiple times.